include/config/config_objects.h

```diff
@@ -73,3 +73,13 @@
 
 // The number of segments Wiggler has, not including the head. Vanilla is 4.
 #define WIGGLER_NUM_SEGMENTS     4
+
+// -- JRB MIST --
+
+// Makes the JRB mist always appear on act 1.
+#define JRB_ACT_SPECIFIC_MIST
```

Comment:
These new defines should probably be off by default

Reply:
Done